src/main/resources/assets/js/resource/LicenseRequest.ts

```diff
@@ -0,0 +1,20 @@
+import {Path} from 'lib-admin-ui/rest/Path';
+import {ResourceRequest} from 'lib-admin-ui/rest/ResourceRequest';
+
+export abstract class LicenseRequest<T>
+    extends ResourceRequest<T> {
+
+    protected constructor() {
+        super();
+    }
+
+    protected generateUrl(): string {
+        // that makes it work from both studio and studio.plus context
+        const idToReplace: string = CONFIG.appId.replace('.plus', '');
+        return CONFIG.services.i18nUrl.replace(new RegExp(`${idToReplace}/`, 'g'), `${idToReplace}.plus/`).replace('i18n', 'license');
+    }
```

Comment:
This is an ugly hack.
We should simply move the license service to the Content Studio app. Remember that you can provide `application` to `serviceUrl` if you need access to a service in a different application. So let's please have proper service Url here.